controls/SV-221584.rb

```diff
@@ -21,4 +21,12 @@
   tag legacy: ["SV-57639","V-44805"]
   tag cci: ["CCI-002605"]
   tag nist: ["SI-2 c"]
+
+  domain_role = command('(Get-Item (Get-ItemProperty \'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe\').\'(Default)\').VersionInfo.ProductVersion')
+
+  google_version = input('74.0.0')
```

Comment:
You can use inputs like in this example. https://mitre-saf-training.netlify.app/courses/beginner/05.html#refactoring-the-code-to-use-inputs Let me know if you have any questions on this!

Reply:
I see you have already implemented `sensitive_system` in that way. I would recommend changing the google version to do the same pattern

Reply:
Keeping this a bit simplier - is there a simple `/v` or `--version` on the shell we could use?

Reply:
`chome.exe /v` or `chome.exe --version` for example? and then we just strip the responce?

Reply:
That doesn't seem to work for me either

Reply:
I was just guessing if the chrome.exe had a version flag, aka suggesting another path of attack to get the info